### DISCUSSION TITLE
Welcome to Oppia!! 👋

### DISCUSSION THREAD

**Discussion by gp201** [oppia/oppia-android#4788] [Developer Announcements]
# Welcome to the Oppia Community :wave: :sparkles:

Hi devs, welcome to Oppia!!

Thank you for joining us in our quest to make high-quality education a human right rather than a privilege.

### Check out these quick tips to get started:

**:bookmark: Bookmark GitHub Discussions**
We're using GitHub Discussions as a place to connect with other members of our community. This is a one-stop shop for all your questions, announcements, and contacting other Oppia members!!

We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together :muscle: 

Feel free to explore the other channels and join in the discussion!

[**How do I contribute?**](https://github.com/oppia/oppia-android/discussions/4094)
There are many ways you can contribute to Oppia. We recommend you go to this [post](https://github.com/oppia/oppia-android/discussions/4094) to see how to get started and contribute to Oppia!

[**How do I get help?**](https://github.com/oppia/oppia-android/discussions/4789)
If you're stuck on a problem, you can always ask the Oppia community for help in the GitHub Discussions. You can read more about [how to get help here](https://github.com/oppia/oppia-android/discussions/4789)

If you have any questions feel free to contact the @oppia/android-welfare-team for assistance.

> **Note**
> In general, please use GitHub Discussions for communication and do not send personal emails or IMs to mentors (unless the issue you're writing about is sensitive in some way). We prefer to conduct introductions/discussions in public channels so that everyone can benefit from the responses. Thanks for your understanding.

**Comment by iKrNitin** [comment]
Hii I am Nitin Kumar

**Reply by dev-divyansh** [reply]
Hello Divyansh here

**Comment by SidharthMudgil** [comment]
Hi, I am Sidharth Mudgil

**Comment by Iamlooker** [comment]
Yo

**Comment by jkugsiya** [comment]
Hi there! I am Jayesh kugsiya, I have contributed to Oppia-android repo before, but its been a long time and I want to join back for contribution. I am looking for starting issues to get re-started.

**Comment by predator313** [comment]
Hi Aamir here

**Comment by ayush290104** [comment]
Hi, Ayush, here I wish to start with open-source contributions in Oppia android..

**Reply by adhiamboperes** [reply]
Hi @ayush290104, have you signed the CLA and set up the project on your machine?

**Reply by ayush290104** [reply]
yes @adhiamboperes I did fork the project and even pointed out an issue that is issue #5106

**Reply by ayush290104** [reply]
Can you tell me more about CLA

**Reply by adhiamboperes** [reply]
Here is the [link](https://goo.gl/forms/AttNH80OV0). This is an agreement between you and Oppia that you are authorised to distribute the code that you submit to Oppia, and that they are your original creations. The license details are included in the form, and you agree to it by filling the mandatory fields.

**Comment by predator313** [comment]
Yes all done but got no response from oppia

**Reply by adhiamboperes** [reply]
@predator313, @ayush290104  and everyone else, in order to get assigned to an issue, go to this [link](https://github.com/oppia/oppia-android/labels/good%20first%20issue)  and review the issues that do not have anyone in the `assignee` section. If you find something interesting, leave a comment describing how you think the issue can be solved (files to be changed, etc) or ask for clarification. An Oppia team member will review your proposed solution and once a good solution has been discussed, you can be assigned to that issue.

**Comment by predator313** [comment]
can any  one please provide updated
gradles
I am not able to run the project in android studio

**Reply by kkmurerwa** [reply]
Please check out the [Wiki](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-Android). It should help you set up the project and run it the first time. Feel free to reach out if you encounter any problems after that.

**Comment by predator313** [comment]
i follow the every steps
but still not able to run the project successfully
please help me
gradles failed

**Reply by adhiamboperes** [reply]
Could you please open a new discussion thread with the specific issue you are facing, including the stacktrace

**Comment by sichchurov** [comment]
Hello everyone! My name is Sergei. I'm glad to join to you!

**Reply by sichchurov** [reply]
[According to this instruction](https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#3-pick-your-first-starter-issue)
I selected the first launch issue, left a comment and sent a PR. Should I have sent a PR or waited for a response from the team lead?

**Reply by adhiamboperes** [reply]
@sichchurov, you can always put up a draft PR to demonstrate your idea.

**Comment by deonwaju** [comment]
Hello Gideon here.

**Comment by aryanmishra29** [comment]
Hello, I am Aryan, an Android Developer. I wish to contribute to the project.

**Reply by Rd4dev** [reply]
Hi Aryan,
Welcome aboard! We're thrilled to have you join as an Android Developer and contribute to the project. Feel free to dive in, explore, and don't hesitate to ask any questions whenever you encounter an issue.

**Comment by prateek2pathak** [comment]
Hello, I am Prateek and I want to contribute to Oppia Android.

**Reply by Rd4dev** [reply]
Welcome, @prateek2pathak! We're thrilled to have you contribute to Oppia Android. 
You can start off by reading through the [Contributors' wiki](https://github.com/oppia/oppia-android/wiki).

**Comment by beradeep** [comment]
Hi, I am Subhradeep. I am interested in Android and Open-Source and glad to join y'all. Looking forward to contribute to Oppia Android.

**Reply by Rd4dev** [reply]
Hi @beradeep, we're delighted to welcome you to our community!

**Comment by Hariomkantsharma** [comment]
Hi! I am Hariom. I am an Android developer. I am looking forward to starting my open-source contribution with Oppia Android.

**Reply by Rd4dev** [reply]
Welcome @Hariomkantsharma, we're thrilled to have you join us.

**Comment by Samarthpatki** [comment]
Hey , I am Samarth , looking forward to contribute to oppia android !

**Reply by Rd4dev** [reply]
Hi @Samarthpatki, we're delighted to welcome you to our community!

**Comment by Photon3009** [comment]
Hey, I am Shivam, an Android developer, would love to contribute to Oppia!

**Reply by Rd4dev** [reply]
Hi Shivam! Welcome to the Oppia community. Check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started on your journey. If you have any questions, feel free to ask. Happy coding!

**Reply by Photon3009** [reply]
Sure, thanks!

**Comment by sahil19solanki** [comment]
Hello everyone

**Reply by Rd4dev** [reply]
Hey @sahil19solanki, welcome to the Oppia community!

**Comment by cybersokari** [comment]
Hello, everyone; I am Sokari, a mobile app developer. I am looking forward to adding my contributions to this project.

**Reply by Rd4dev** [reply]
Hi Sokari! Welcome to the Oppia community. Check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started on your journey. If you have any questions, feel free to ask. Happy coding!

**Comment by subhajitxyz** [comment]
Hello everyone, I am Subhajit Mallick. I am an android app developer and looking forward to contribute in Oppia.

**Reply by Rd4dev** [reply]
Hi @subhajitxyz, Welcome to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started, and feel free to reach out if you have any questions. Happy coding!

**Comment by jainv4156** [comment]
Hello everyone , I am Vaibhav Jain . Just started  android with kotlin and having fun contributing to open source

**Reply by adhiamboperes** [reply]
Hi @jainv4156, Welcome to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) on to get started setting up the project, and feel free to reach out via discussions if you have any questions. Happy coding!

**Reply by jainv4156** [reply]
sure

**Comment by TanishMoral11** [comment]
Hi everyone,

I'm Tanish Moral from India. I have a passion for developing Android apps and contributing to open-source projects. I recently started exploring Oppia about a week ago and have enjoyed the journey so far.

I've submitted a PR ( https://github.com/oppia/oppia-android/pull/5555 ) that addresses "the splash screen issue"] but unfortunately, it failed some CI checks. I'm not entirely sure how to resolve these issues, and I would be grateful if someone could guide me through the process.

**Reply by Rd4dev** [reply]
Hi @TanishMoral11, welcome to Oppia Android!

Regarding the CI checks, initially those workflows require approval from a maintainer before they can run. Once approved, the CI checks will be triggered. The current status of those checks isn't indicative of any issues—they haven't run yet, and you'll only need to address concerns (if any) once the CI process is completed.

Please ensure you've signed the CLA form (if you haven't already), and if your PR is ready for review, you can [assign reviewers](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section) by tagging them, as ['@ReviewerName PTAL'.](https://github.com/oppia/oppia-android/pull/5553#issuecomment-2411080069) They'll take a look and help move the process forward, including running the CI.

**Comment by manas-yu** [comment]
Hi everyone! I’m Manas, and I’ve recently started exploring Oppia Android. I’m really excited to dive deeper into the project, discuss issues, and contribute to making it even better! 🚀 Looking forward to collaborating with you all.

**Comment by KaranJoshi1208** [comment]
Hi,
I’m Karan, a 2nd-year BTech student and an Android developer. Excited to learn, explore, and connect with you all!

**Reply by adhiamboperes** [reply]
Hi @KaranJoshi1208, Welcome to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started, and feel free to reach out if you have any questions. Happy coding!

**Reply by KaranJoshi1208** [reply]
Thankyou, @adhiamboperes

**Comment by sonukumar005** [comment]
Hii ,I am Sonu Kumar,and really excited to work with oppia

**Reply by adhiamboperes** [reply]
Hi @sonukumar005, thank you for your interest in contributing to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started, and feel free to reach out if you have any questions. Happy coding!

**Comment by angrezichatterbox** [comment]
Hi, I am Gautham Mohanraj, I am interested to contribute to oppia android

**Comment by siddhisanap** [comment]
Hi,
I am Siddhi Sanap, i am new to open source but i am really intrested to contribute to oppia 
i hope i have enough skills and will be of use to the community. please give me a chance to prove myself to this community

**Reply by adhiamboperes** [reply]
Hi @siddhisanap, thank you for your interest in contributing to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started, and feel free to reach out if you have any questions. Happy coding!

**Comment by joelmuraguri** [comment]
Hi I am Joel. Happy to join you

**Reply by adhiamboperes** [reply]
Hi @joelmuraguri! Welcome to Oppia!

**Comment by siddhisanap** [comment]
hello can u please guide me what do i have to do here because i am not able to get i!

**Reply by TanishMoral11** [reply]
Hey @siddhisanap, Welcome to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started, 
Also, you can pick an issue from the [Good First Issue](https://github.com/oppia/oppia-android/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label to start contributing.
and feel free to reach out if you have any questions. Happy coding!

**Comment by aadityaverma2011** [comment]
Supp everyone! Aaditya Verma here , an aspiring developer  and i just wanna let y'all know that i have spent a considerable amount of time in the past 3 days in researching behind the Oppia Foundation (and setting up that android Codebase 😉 ) and to be very honest i am actually inspired by the idea. Anyways i am gonna really commit myself to the development of this project atleast for a few months from here on , fixing up or adding up whatever i can !

**Reply by subhajitxyz** [reply]
Hi @aadityaverma2011, Welcome to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started, and feel free to reach out if you have any questions. Happy coding!

**Comment by suyashshenoi** [comment]
Hi everyone,
I had previously worked on a small A11Y issue, and my PR for that was merged. Due to personal reasons, I had to put contributing on hold for a while, but I'm excited to be back now! Looking forward to picking up new issues, learning from the community, and making meaningful contributions again.

**Comment by Santhosh-Paramasivam** [comment]
Hello everyone!! Just wanted to say hi, very excited to be here:)

**Comment by Mia0451** [comment]
Hi everyone, my name is Mia Shi, I am so happy to join the discussion here! I am very interested in contributing to the android project.

**Comment by Amit-Matth** [comment]
Hi, I'm Amit Matth, and I'm excited to contribute to Oppia-Android!

**Comment by Akash05-kr** [comment]
Hi everyone! 👋
I’m Akash Kumar Chaurasiya, I’m passionate about Android development and have experience working with Kotlin.
I’m really excited to be part of the Oppia Android community and to collaborate, learn, and contribute to open-source projects. Looking forward to connecting with you all and growing together! 😊

**Reply by TanishMoral11** [reply]
Hey @Akash05-kr , Welcome to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started, 
Also, you can pick an issue from the [Good First Issue](https://github.com/oppia/oppia-android/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label to start contributing.
and feel free to reach out if you have any questions. Happy coding!

**Comment by itsadityaaaaa** [comment]
Hi everyone! 👋
I am Aditya Verma a aspiring developer intrested in  Oppia android development looking forward to collaborate with you all to do some impactful contributions in the project.

**Reply by subhajitxyz** [reply]
Hey @itsadityaaaaa , Welcome to Oppia Android! Please check out the [contributor's wiki](https://github.com/oppia/oppia-android/wiki) to get started,
Also, you can pick an issue from the [Good First Issue](https://github.com/oppia/oppia-android/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label to start contributing.
and feel free to reach out if you have any questions. Happy coding!

**Comment by shrimpnaur** [comment]
Hello everyone, this is Shria Nair here. I am very much interested to contribute to oppia-android as I found this org pretty welcoming to new-comers and has a really detailed wiki about everything. Looking forward to contribute more and learn along the way and also aiming for gsoc next year :)

**Reply by manas-yu** [reply]
Hi @shrimpnaur ! Welcome to the Oppia community, great to have you here! It's great that you took a look at the wiki section. You can follow the [Installing Oppia Android](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android) guide to set up the project and explore some [good first issues](https://github.com/oppia/oppia-android/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22good%20first%20issue%22%20%20no%3Aassignee). Awesome to hear you're aiming for GSoC — feel free to reach out if you have any questions!

**Comment by iAbanoubSamir** [comment]
Hi everyone 👋, its Abanoub.
Looking forward to collaborate with you all to build a great stuff.

**Comment by theayushyadav11** [comment]
Hi @iAbanoubSamir ! Welcome to Oppia community.
Here are also some of the issues mentioned in the [discussion](https://github.com/oppia/oppia-android/discussions/5850) that would match your interest.
Happy coding.

**Comment by crlsribeiro** [comment]
Hello everyone, my name is Carlos Ribeiro, I'm joining this community which is of unconditional value to me, I'm very happy to be able to contribute.

Thank you very much!

**Reply by subhajitxyz** [reply]
Welcome, @carlosaoribeiro !  We're thrilled to have you contribute to Oppia Android.
You can start off by reading through the [Contributors' wiki](https://github.com/oppia/oppia-android/wiki).

Here are also some of the issues mentioned in the [discussion](https://github.com/oppia/oppia-android/discussions/5852) that would match your interest.Happy coding!

**Comment by adhiamboperes** [comment]
Hi @rezaul-web, please feel free to select some [good first issues](https://github.com/oppia/oppia-android/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22good%20first%20issue%22%20%20no%3Aassignee) after completing the setup instructions.

**Comment by EngDann** [comment]
Hi everyone!

My name is Danilo de Melo, and I'm a Software Engineer with over three years of experience, primarily focused on mobile and backend development. I'm also currently a student at the University of Brasília.

I chose Oppia because I was looking for an open-source project with a meaningful cause that aligns with my skills. I truly love the idea of contributing to making education a human right.

I'll be starting my contribution as part of a university subject requirement, and I am also deeply interested in applying for GSoC with Oppia.

**Reply by adhiamboperes** [reply]
Hi @EngDann,

Thanks for your interest in contributing to Oppia Android! Looking forward to working with you.

**Comment by thevedantchourey** [comment]
Hi Oppia team! 👋

My name is Vedant Chourey, and I’m a final-year B.Tech Computer Science student.
I’ve been developing Android apps using Kotlin, Jetpack Compose, and Firebase.
I’m very excited to start contributing to Oppia Android, learn from experienced mentors, and eventually apply for GSoC 2026 with Oppia.

Glad to be here — and I’ll try to learn with every bug, feature, and challenge! 💪

My first freelance project is live on the Play Store — please do check it out:
👉 [ApplyWisee](https://play.google.com/store/apps/details?id=com.app.applywisee&pcampaignid=web_share)

**Reply by manas-yu** [reply]
Hi Vedant! Welcome to Oppia!
Great to have you here. Excited to see your contributions and GSoC journey ahead. ApplyWisee looks great too!
You can get started by checking out [Wiki](https://github.com/oppia/oppia-android/wiki) and browsing [Good First issues](https://github.com/oppia/oppia-android/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22good%20first%20issue%22%20%20no%3Aassignee). Happy coding!

**Comment by thevedantchourey** [comment]
Thanks! 👋

I’m excited to be here — thanks for the warm welcome. I’ve set up the Oppia Android project locally and am looking for a good first issue to start with. If anyone recommends a beginner-friendly Android issue or task I can take on, I’d really appreciate the guidance.

I’m comfortable with Kotlin, Jetpack Compose, and Android Studio — and would be happy to help with UI/UX, Jetpack Compose, Firebase, third-party library integrations, or small logic fixes.

Please let me know, if there’s any beginner-friendly Android issue I can pick up to get started. 🙂

**Reply by adhiamboperes** [reply]
Hi @thevedantchourey, please take a look at our recommended starter issues: https://github.com/oppia/oppia-android/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22%20no%3Aassignee

**Comment by CodeWithSangeeta** [comment]
Hi everyone 👋, I’m Sangeeta Yadav!

I’m final year BCA Student and learning Android Development, mainly using Kotlin, Jetpack Compose, Room Database , Firebase and Android Studio. I recently completed setting up Oppia Android with Bazel and WSL, and I’m excited to start contributing.

I’m passionate about open source and want to improve my development skills while collaborating with others. I’m also preparing for Google Summer of Code, and Oppia’s mission and codebase really inspired me.

I look forward to contributing to the Android project, fixing beginner-friendly issues, improving features, and learning from the community.
If there are any first tasks or recommended starting points, I would be grateful for guidance. 😊

Thanks & excited to be part of Oppia!

— Sangeeta ✨

**Reply by subhajitxyz** [reply]
Hi Sangeeta! Welcome to Oppia! 
It’s great to hear that you’ve completed the setup. Excited to see your contributions at oppia.
You can get started by checking out [Wiki](https://github.com/oppia/oppia-android/wiki) and browsing [Good First issues](https://github.com/oppia/oppia-android/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22good%20first%20issue%22%20%20no%3Aassignee). Happy coding!

**Reply by CodeWithSangeeta** [reply]
Thank you so much! 😊
I’ve started exploring the Wiki and the Good First Issues as suggested.
I also completed my setup and have begun understanding the codebase.

Excited to learn more and contribute to Oppia!

**Reply by simrithi** [reply]
Hey Sangeeta Didi, I am a second-year BTech CSE Student from Chennai. I am interested in a similar tech stack as you, glad to work with you on this repo :)

**Reply by CodeWithSangeeta** [reply]
Hi @simrithi! 
So nice to meet you! I’m also quite new to open-source and still learning the Oppia codebase step by step.
Would love to collaborate and learn together. 
Feel free to reach out if you want to discuss anything or explore issues together!

**Reply by simrithi** [reply]
Hey can we have a discussion regarding the issues? Sorry I have my internship going on so could not catch up with you

**Comment by simrithi** [comment]
Hello All! I am Simrithi S from Chennai ,India. I am highly excited to contribute to Oppia Foundation and make a great impact!. Glad to work with everyone and always feel free to contact me at my mail: simrithi.s@gmail.com. Thanks!:)

**Reply by adhiamboperes** [reply]
Hi @simrithi, thanks for your interest in contributing to Oppia Android. First, set up your developer environment by following the instructions in the [Wiki](https://github.com/oppia/oppia-android/wiki). Next, select an issue from one of these - https://github.com/oppia/oppia-android/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22%20no%3Aassignee. 

Please let us know if you run into issues setting up, or need help choosing an issue to work on.

**Reply by simrithi** [reply]
Thank you 😊 sure !

**Comment by simrithi** [comment]
Thank you sure!

**Comment by geekPastor** [comment]
Hello 👋,
I hope you're doing well.

My name is Chrinovic Mukeba, a Junior Android Developer experienced with Kotlin, Jetpack Compose, MVVM, Firebase, and REST API integration. I’m looking to start contributing to open-source projects, and I’m really interested in the work you’re doing on this project.

As this would be my first open-source contribution, I’d love to help with:

fixing beginner-friendly issues,

improving UI components or code structure,

adding documentation,

or any task you think would be a good starting point.


I’m motivated, open to learning, and ready to contribute consistently.
Please let me know how I can get started or if there’s a good issue for me to pick up.

Thank you! 🙌

**Comment by mahakaal2005** [comment]
Hi everyone!
I’m Atul, an Android developer from KIET (India). I mostly work with Kotlin and Android app development, but I’m excited to learn more about Python, modern web technologies, and contribute to a meaningful open-source project like Oppia.

I’m here to grow, collaborate, and hopefully take on some impactful features. Looking forward to learning from the community and contributing wherever I can!

**Reply by adhiamboperes** [reply]
Hi @mahakaal2005! Welcome to Oppia, we are please to have you on board. Please read the [Wiki](https://github.com/oppia/oppia-android/wiki) set up guide, and then select an issue from one of these - https://github.com/oppia/oppia-android/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22%20no%3Aassignee to work on. 

Please let us know if you are stuck at any point.

**Reply by mahakaal2005** [reply]
Hey @adhiamboperes! Sorry to be so late ,I had my exams now i will set things up and look up for the issues.
Really excited to dive deep into the code!!!

**Comment by ayush2006-creator** [comment]
Hi everyone! I’m Ayush, a Computer Science undergraduate at NIT Jalandhar and an Android developer working primarily with Kotlin. I’m interested in contributing to Oppia Android. Looking forward to learning from the community and collaborating!

**Comment by Kishan8548** [comment]
Hi everyone!

I'm Kishan, and I'm interested in Android development and open-source contributions.

I’ve been learning Kotlin and exploring how large-scale Android projects are built and maintained. Recently, I opened my first PR to Oppia Android — PR #6254.

Excited to learn more, contribute to the project, and collaborate with the community.

**Reply by adhiamboperes** [reply]
Hi Kishan, welcome to Oppia and thanks for your initial contribution!

I have looked at PR https://github.com/oppia/oppia-android/pull/6254. You have received an initial review and you are expected to address and respond to the comments, and assign back to Neer. Please do that, thanks!

**Reply by Kishan8548** [reply]
Hi @adhiamboperes! Apologies for the late response — I had ongoing exams. I've now read through Neer-rn's review on PR #6254 and I'm working on addressing the feedback. I'll assign the PR back to Neer once the fixes are pushed. Thanks!

**Comment by omkodes** [comment]
Hi everyone! 

I'm Omkar Sanap, a B.Tech student at IIT (ISM) Dhanbad '28 and an Android developer working mainly with Kotlin and Flutter. Oppia's mission of making education a human right really resonates with me and I'm thrilled to be part of this community.

I've got the project set up locally and I'm eager to start making meaningful contributions. Excited to learn, grow, and build something that truly matters alongside all of you!

**Reply by adhiamboperes** [reply]
Hi @omkodes, Welcome to Oppia, we are please to have you on board. Please look at these issues, and leave a comment with a brief explanation on which one you would like to work on - https://github.com/oppia/oppia-android/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22%20no%3Aassignee to work on.

Please let us know if you are stuck at any point.

**Comment by r0hansaxena** [comment]
Hello everyone!

I'm Rohan Saxena, I love open source and I've been wanting to get into Android Development and came across Oppia.
I genuinely love what Oppia stands for. I've taught students through an NGO and tuitions, making good education accessible means a lot to me. I'm looking forward to making meaningful contributions and getting hands-on with Android dev!
I've recently set up the project locally and also went through the Oppia dashboard, made a lot of things so much more convenient :)

Really excited to work alongside the community here!

**Reply by Neer-rn** [reply]
Hi @r0hansaxena, welcome to Oppia! It’s great to hear that you’ve completed the setup and explored the app.
Please take a look at our [good first issues](https://github.com/oppia/oppia-android/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22%20no%3Aassignee) and leave a comment on an issue you’d like to work on, including a brief outline of your proposed approach. Feel free to ask if you need help choosing an issue or get stuck. Looking forward to your contributions!
